Fitness/graphql/gymClassInstanceQueries.graphql

```diff
@@ -10,18 +9,20 @@ query AllClassesInstances
         id
         name
         description
-        tags
+          tags {
+              label
+          }
       }
+      imageUrl
```

Comment:
Is backend adding this attribute? I thought they got rid of it with the new backend

Reply:
Yes, the attribute is added